index.html

```diff
@@ -1,15 +1,166 @@
+                    <li>Создание собственной страницы.</li>
+                </ul>
+            </li>
+            <li>Погружение в HTML-разметку
+                <ul>
+                    <li>Мета-теги.</li>
+                    <li>Мнемоники.</li>
+                    <li>Тег для работы с изображениями.</li>
+                    <li>Теги для работы со списками.</li>
+                    <li>Теги для работы с таблицами.</li>
+                    <li>Теги для работы с формами.</li>
+                    <li>Div-блоки.</li>
+                </ul>
+            </li>
+        </ol>
+    </p>
+    <p>
+        <table border="1" width="800px">
+            <tr>
+                <th>Tag</th>
+                <th>Description</th>
+            </tr>
+            <tr>
+                <td>&lt;table&gt;</td>
+                <td>Defines a table</td>
+            </tr>
+            <tr>
+                <td>&lt;tr&gt;</td>
+                <td>Defines a row in a table</td>
+            </tr>
+            <tr>
+                <td>&lt;th&gt;</td>
+                <td rowspan="2">Defines a "header cell" / "cell" in a table</td>
+            </tr>
+            <tr>
+                <td>&lt;td&gt;</td>
+            </tr>
+            <tr>
+                <td colspan="2">Rarely used</td>
+            </tr>
+            <tr>
+                <td>&lt;caption&gt;</td>
+                <td>Defines a table caption</td>
+            </tr>
+            <tr>
+                <td>&lt;thead&gt;</td>
+                <td>Groups the header content in a table</td>
+            </tr>
+            <tr>
+                <td>&lt;tbody&gt;</td>
+                <td>Groups the body content in a table</td>
+            </tr>
+            <tr>
+                <td>&lt;tfoot&gt;</td>
+                <td>Groups the footer content in a table</td>
+            </tr>
+        </table>
+    </p>
+    <p>
+        <form action="/" method="GET">
+            <fieldset>
+                <legend>My first form</legend>
+                <label for="username">Name</label>
+                <br />
+                <input type="text" name="username" id="username" placeholder="Name" required />
+                <br />
+                <label for="email">Email</label>
+                <br />
+                <input type="email" name="email" id="email" required />
+                <br />
+                <label for="password">Password</label>
+                <br />
+                <input type="password" name="password" id="password" required />
+                <br />
+                <label for="lever">Your programming level</label>
+                <br />
+                <input type="range" name="level" id="level" min="1" max="10" />
+                <br />
+                <br />
+                <label for="html">What does HTML stand for?</label>
+                <br />
+                <br />
+                <input type="checkbox" name="html" id=""/>Hyperlinks and Text Markup Language
```

Comment:
Text for input should be wrapped on `label`.

Reply:
Fixed bug with wrapping